src/components/ReviewForm.tsx

```diff
@@ -93,6 +102,11 @@ const ReviewForm = ({
 
   const { setAlert } = useAlert();
   const [userReviews, setUserReviews] = useState<TUserReviews>({});
+  const [difficultyValue, setDifficultyValue] = useState<TRatingScale | null>(
```

Comment:
In `globals/types.ts` we have a type defined called `TNullable` which aliases `x | T` (e.g., `useState<TNullable<TRatingScale>>(...)` here, rather than `useState<TRating | null>(...)`)

Reply:
Okay thanks for the heads up. I'll take a look and fix it